docs/reference/cluster/health.asciidoc

```diff
@@ -131,23 +131,26 @@ include::{es-ref-dir}/rest-api/common-parms.asciidoc[tag=cluster-health-status]
 `unassigned_shards`::
     (integer) The number of shards that are not allocated.
 
+`unassigned_primary_shards`::
+    (integer) The number of shards that are primary but not allocated. **Note**: This number may be lower than the true value if your cluster contains nodes running a version below v8.16. For a more accurate count in this scenario, please use the <<cluster-health,cluster health API>>.
```

Comment:
A small suggestion to be inline with the language in the docs. After a quick look I saw that we do not use the prefix `v`, we just use the version number.
 
```suggestion
    (integer) The number of shards that are primary but not allocated. **Note**: This number may be lower than the true value if your cluster contains nodes running a version prior to 8.16. For a more accurate count in this scenario, please use the <<cluster-health,cluster health API>>.
```

Reply:
Sure thing. Have pushed that change :-)